getting_started/access_your_services.adoc

```diff
@@ -215,6 +335,8 @@ xref:../admin_guide/quota.adoc#admin-guide-quota[managing
 quotas] and xref:../admin_guide/service_accounts.adoc#admin-guide-service-accounts[configuring service
 accounts].
 
+// TODO Need to confirm if the following paragraph is true for ARHO
```

Comment:
On OSA all clusters are configured using the NetworkPolicy SDN.
Project admins can set NetworkPolicy objects on their projects.
Cluster admins can also set EgressNetworkPolicies